### PR TITLE
Changed Family Inheritance Helper to XML

### PR DESCRIPTION
**Pull request by dscarpac**
image

### CODE REVIEW

File: src/playercards/cards/FamilyInheritance.ttslua [hunk 1]
```diff
@@ -51,22 +48,35 @@ function add4(playerColor)
...
   end
-  printToColor("Moved " .. foundTokens .. " resource(s) to " .. matColor .. "'s resource pool.", playerColor)
+  printToColor("Moved " .. foundTokens .. " resource(s) to " .. matColor .. "'s resource pool.", player.color)
 end
 
-function loseAll(playerColor)
+function loseAll(player)
   searchSelf()
-
+  log(player.color)
```
Comment (Chr1Z93) [inline src/playercards/cards/FamilyInheritance.ttslua:64]: Leftover log statement

File: xml/playercards/FamilyInheritance.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<Defaults>
+  <Button padding="30 30 30 30"
+    font="font_teutonic-arkham"
+    textColor = "white"
```
Comment (Chr1Z93) [inline xml/playercards/FamilyInheritance.xml:4]: Extra whitespace (also multiple times below)

File: xml/playercards/FamilyInheritance.xml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    rotation="0 0 270"
+    height="460"
+    width="2600"
+    scale="0.1 0.1 1"
+    cellSpacing="80"
+    cellBackgroundColor="rgba(1,1,1,0)"/>
+</Defaults>
+
+<TableLayout id="Helper"
+active="false">
```
Comment (Chr1Z93) [inline xml/playercards/FamilyInheritance.xml:18]: Missing indentation

**Review by Chr1Z93 — CHANGES_REQUESTED** [review]

File: xml/playercards/FamilyInheritance.xml (hunk 2 above)
Comment (dscarpac) [reply xml/playercards/FamilyInheritance.xml:4]: I'm not seeing it for some reason?

File: xml/playercards/FamilyInheritance.xml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        text="Discard all"/>
+    </Cell>
+    <Cell>
+      <Button onClick="takeAll"
+        color="#173B0B"
+        text="Move all"/>
+    </Cell>
+    <Cell>
+      <Button onClick="add4"
+        color = "#77674D"
```
Comment (Chr1Z93) [inline xml/playercards/FamilyInheritance.xml:33]: Whitespace next to equal sign

File: src/playercards/cards/FamilyInheritance.ttslua [hunk 5]
```diff
@@ -51,22 +48,33 @@ function add4(playerColor)
...
+function shutOff()
+  self.UI.hide("Helper")
+  updateSave()
+end
+
+-- show buttons
+function initialize()
+  self.UI.show("Helper")
+  updateSave()
+end
```
Comment (Chr1Z93) [inline src/playercards/cards/FamilyInheritance.ttslua:80]: Missing new line at the end of file

File: src/playercards/cards/FamilyInheritance.ttslua [hunk 6]
```diff
@@ -1,26 +1,23 @@
...
 local playermatApi = require("playermat/PlayermatApi")
 local searchLib = require("util/SearchLib")
 local tokenManager = require("core/token/TokenManager")
 
+local isHelperEnabled      = false
 local clickableResourceCounter = nil
 local foundTokens = 0
 
+function updateSave()
+self.script_state = JSON.encode({ isHelperEnabled = isHelperEnabled })
```
Comment (Chr1Z93) [inline src/playercards/cards/FamilyInheritance.ttslua:11]: Missing indentation

**Review by Chr1Z93 — CHANGES_REQUESTED** [review]

**Review by Chr1Z93 — APPROVED** [review]
GitHub still says that one file is missing a end of file new line, but I'll let that pass 😉
Thanks for the flashy new XML!

**Pull request merged**